docs_source_files/content/webdriver/page_loading_strategy.en.md

```diff
@@ -244,7 +244,14 @@ public class pageLoadStrategy {
 }
   {{< / code-panel >}}
   {{< code-panel language="python" >}}
-# Please raise a PR
+from selenium import webdriver
+from selenium.webdriver.common.desired_capabilities import DesiredCapabilities
```

Comment:
Hi @abhishek-malani ,

Thanks for the PR. Apologies, but I'v couple of things to discuss here
1. Code sample didn't work for me when I tested with selenium alpha bindings.
2. We're moving away from `desired capabilities`, please use `chromeOptions` to set the capability
2. Can you please make code changes to other translated pages too.

Regards,
Harsha